### PR TITLE
Merge the existing Support Docs (Redmine Docs) into the SecureDrop Documentation

### LINKED ISSUE #471: Merge existing content from redmine_docs into securedrop-docs
- Merge existing git history from redmine_docs into securedrop-docs (conversation with @legoktm to come)
- update nav etc to reflect new content location

### PR DESCRIPTION
**Pull request by nathandyer**
## Status

Ready for Review

## Description of Changes

This PR merges the git history from the Redmine docs repository and merges the content from the previous repository into the "Getting Support" article that is part of the main SecureDrop documentation.

* Resolves #471 

## Testing
- [x] Git history is in tact
- [ ] Content in the "Getting Support" page matches the relevant information from the old Redmine docs portal
- [x] CI passes
- [x] Visual inspection

## Release 

Prior to merging, we'll want to make sure that there are redirects in place from the old Redmine docs website pointing to this support article.

Once it is merged, we'll need to tag a stable release ASAP.

## Checklist (Optional)

- [x] Doc linting (`make docs-lint`) passed locally
- [x] Doc link linting (`make docs-linkcheck`) passed
- [x] You have previewed (`make docs`) docs at http://localhost:8000

### CODE REVIEW

**Comment by eloquence** [comment]
Nathan, just a note that the most recent commits are unsigned.

**Comment by eloquence** [comment]
Overall I think this is coming together nicely. I didn't spot an issue with the commit history, but Kunal's trained eyes will be definitely helpful there.

I'm assuming this is still WIP since it's marked as such; let me know when it's ready for formal review. One suggestion: With all this new content, I think we should add a short intro (spelling out the two main support paths and linking to the appropriate subsections).

**Comment by nathandyer** [comment]
Thank you @eloquence! The commits should be signed and displaying properly here, now. I also added a brief introduction and linked out to some of the sections below, as you recommended.

I think this is officially ready for review whenever someone has time to look at it. Thanks again!

File: docs/getting_support.rst
Comment (eloquence) [inline docs/getting_support.rst:294]: This feels a bit sandwiched between Redmine stuff right now, I would suggest moving it to the end or the beginning of the chapter.

File: docs/getting_support.rst
Comment (eloquence) [inline docs/getting_support.rst:310]: I think we'll need to tweak organization and section title here. What are we troubleshooting - Redmine access itself, or SecureDrop? If it's the latter, some of the sub-sections don't really fit. Given that this is a second-level heading, I could see folks getting confused -- "I'm trying to troubleshoot 2FA on SecureDrop, but this seems to be something else entirely"

The old troubleshooting section was always messy and will ultimately benefit from some cleanup. For now, I would suggest:
- Put critical onboarding and usage information in the first chapter (i.e. creating issues, setting up 2FA)
- Put support portal tips and troubleshooting information in the second chapter, under a title like "Support Portal Tips and Troubleshooting" or similar, that leaves now doubt about its purpose

File: docs/getting_support.rst
Comment (nathandyer) [reply docs/getting_support.rst]: Agreed - I decided to move it to the end of the chapter, to encourage folks to use the support portal over the community support (when it makes sense to do so).

File: docs/getting_support.rst
Comment (nathandyer) [reply docs/getting_support.rst]: Thanks! I ended up restructuring this quite a lot, and in the end I realized the information in the current "Troubleshooting" section actually belonged within the main headings rather than a catch-all "Troubleshooting" section. As a result, as of the most recent commit, this section is now omitted entirely. Please let me know if you like this new structure better, and if you think it's necessary to have tips/troubleshooting section here at all.

File: docs/getting_support.rst
Comment (eloquence) [inline docs/getting_support.rst:179]: I would suggest we make this part of the "creating an account" portion, prior to them testing out ticket submission and such -- this helps establish that 2FA is part of the account setup process, and not something to maybe pick up later.

**Comment by eloquence** [comment]
One structural suggestion, otherwise LGTM pending Kunal's sign-off on the history merge :)

File: docs/getting_support.rst
Comment (legoktm) [inline docs/getting_support.rst:309]: The advice makes sense (don't bother unless you already know PGP), but as written now it feels unnecessarily personal?

"If you are not already familiar with using PGP/GPG keys,we would recommend using the web-based workflow instead of setting up the encrypted email workflow." or something.

File: docs/getting_support.rst
Comment (legoktm) [inline docs/getting_support.rst:378]: Enigmail is dead, "The default settings for Thunderbird should work."

File: docs/getting_support.rst
Comment (legoktm) [inline docs/getting_support.rst:383]: I think we should either have an explicit "don't use Mailvelope" or remove this aside, it doesn't seem helpful as is.

File: docs/getting_support.rst
Comment (legoktm) [inline docs/getting_support.rst:394]: Just Thunderbird

File: docs/getting_support.rst
Comment (legoktm) [inline docs/getting_support.rst:401]: The menu item is now called "End-to-End Encryption".

AFAICT there's no longer a PGP/MIME option, I think it's the only thing that's supported?

And...I can't figure out how to set a per-recipient rule. If I compose an email to support@ it does prompt me that encryption is possible, but I haven't yet found an option to enforce it.

**Review by legoktm — APPROVED** [review]
Git history merge looks good! I did a quick pass on the content, but I assume we'll fix those in follow-ups since it's not related to the merge itself.

File: docs/getting_support.rst
Comment (nathandyer) [reply docs/getting_support.rst:401]: Given these changes to Thunderbird (and the fact that the options we're flagging are either removed/non-existent), and that we don't provide these details for any other email clients, I'm going to suggest that we remove this Thunderbird-specific section entirely.

**Comment by nathandyer** [comment]
Thanks for the thorough review, @legoktm! I just pushed a revision that I believe addresses your suggestions. I'm going to leave some of the threads open for now in case we want additional discussion, but if my changes look OK to you, please feel free to mark as resolved.

File: docs/getting_support.rst
Comment (legoktm) [reply docs/getting_support.rst]: I think it's probably worth keeping the warning in some form, "Note that Mailvelope does not support PGP/Mime" (AFAICT it still doesn't: https://github.com/mailvelope/mailvelope/issues/41), I just thought the admonishment "you shouldn't be using it anyways" was out of place.

File: docs/getting_support.rst
Comment (legoktm) [reply docs/getting_support.rst]: Yeah I think that's reasonable. If anything, we could just link to https://support.mozilla.org/en-US/kb/openpgp-thunderbird-howto-and-faq and leave it at that.

Now that we're discussing it, I am curious 1) how many people are using PGP email for redmine and 2) what email clients they're using. The former seems like something we could get out of mail server logs or the redmine DB (checking if people have a key set), the latter would require logging incoming mail before redmine processed it? AFAICT Thunderbird sends the `User-Agent` field in cleartext, idk about other clients. Maybe not worth the effort, but I'm at least curious :)

**Comment by legoktm** [comment]
Besides https://github.com/freedomofpress/securedrop-docs/pull/474#discussion_r1271060407, LGTM :)

**Comment by legoktm** [comment]
The only thing I see missing from the existing redmine_docs is this note at the beginning:
> The [support site](https://support.freedom.press/) is also available as a Tor Onion Service at http://sup6h5iyiyenvjkfxbgrjynm5wsgijjoatvnvdgyyi7je3xqm4kh6uqd.onion/.

Was that an intentional omission? In any case it definitely doesn't need to be that prominent of a note, just a parenthetical would be fine if we want to mention it at all.

**Comment by nathandyer** [comment]
@legoktm Yes, that was an intentional omission here. Because this is now a chapter deeper down within the Overview section of the documentation, having it mention the onion version felt a little out of place. 

That does make me realize that we don't a comparable note for the SecureDrop docs at all, which I definitely think we should. With that being the case, I might suggest we merge this as-is, and then add a similar note on the `index` page, pointing folks to `http://dftlffjdogaragaxkc6jqxpo77s7rrngimyoq7uuq3clowhmttblcoyd.onion`

**Comment by legoktm** [comment]
The onion link I mentioned goes to the equivalent of redmine, not the docs. :) Let me put in a suggestion to show what I mean...

And +1 to having a note on the docs main page saying they're also available via an onion service.

**Review by legoktm — APPROVED** [review]
Awesome work!

**Pull request merged**
